linera-sdk/src/util.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//! Module with helper types and functions used by the SDK.
```

Comment:
Nice! Perhaps include a couple of unit tests in the same file?

Reply:
Good idea! Done.